src/WebJobs.Script.Cli/WebJobs.Script.Cli.csproj

```diff
@@ -15,7 +15,7 @@
     <RunCodeAnalysis>true</RunCodeAnalysis>
     <CodeAnalysisRuleSet>..\src.ruleset</CodeAnalysisRuleSet>
     <CodeAnalysisCulture>en-US</CodeAnalysisCulture>
-    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <TreatWarningsAsErrors>false</TreatWarningsAsErrors>
```

Comment:
I see. That being the case, feel free to leave as is. I'll take note and we'll have a chance to look at this again when merging this code into `dev`